nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -260,6 +310,20 @@ protected String getQuery(DatabaseAdapter dbAdapter, String tableName, String co
     }
 
 
+    protected Map<String,String> getDefaultMaxValueProperties(final Map<PropertyDescriptor, String> properties){
+        final Map<String,String> defaultMaxValues = new HashMap<String, String>();
+
+        for (final Map.Entry<PropertyDescriptor, String> entry : properties.entrySet()) {
+            final String key = entry.getKey().getName();
+
+            if(!key.startsWith(INTIIAL_MAX_VALUE_PROP_START)) continue;
```

Comment:
Not sure (yet) if our checkstyle allows the one-line-if, but if you're already in here making changes, could you add the explicit braces? Superfluous I know, but consistent and extensible :)